pygmt/src/sphdistance.py

```diff
@@ -35,6 +35,10 @@ def sphdistance(table, **kwargs):
 
     Parameters
     ----------
+    table : str or {table-like}
```

Comment:
Wait for more votes to come in on whether to use `data` or `table` at https://github.com/GenericMappingTools/pygmt/issues/1479#issuecomment-912203679 :slightly_smiling_face:

```suggestion
    datatable : str or {table-like}
```

Reply:
Voted in #1479 😉

Reply:
Ok, looks like we're going with `data`! Please apply the change.